mlir/include/mlir/Dialect/Rock/Tuning/QuickTuningPerfconfigs.inc

```diff
@@ -1304,17 +1755,25 @@ static const InitParamsAccel initParametersForwardI8ConvGfx1200[nInitParametersF
 {"gfx950_conv_i8", {PopulateParamsXDL::initParametersForwardI8ConvGfx950, PopulateParamsXDL::nInitParametersForwardI8ConvGfx950}},
 {"gfx1000_gemm_f16", {PopulateParamsWmma::initParametersFp16GemmGfx1000, PopulateParamsWmma::nInitParametersFp16GemmGfx1000}},
 {"gfx1100_gemm_f16", {PopulateParamsWmma::initParametersFp16GemmGfx1100, PopulateParamsWmma::nInitParametersFp16GemmGfx1100}},
-{"gfx1200_gemm_f16", {PopulateParamsWmma::initParametersFp16GemmGfx1200, PopulateParamsWmma::nInitParametersFp16GemmGfx1200}},
```

Comment:
Does the gfx1200 fall back to gfx1201?  The gfx1200 does exist and should be supported.

Reply:
Yes, it will fallback to gfx1201. The fallback logic is implemented in https://github.com/ROCm/rocMLIR/pull/1994, and it basically looks for the closest arch if it can't find an exact match. The old list was also based on gfx1201, it was just generically named gfx1200 because it would be used for any gfx12**, that's why it is redundant now.

We can tune for gfx1200 as well, if needed.